packages/expandable/index.js

```diff
@@ -89,9 +89,9 @@ class WarpExpandable extends kebabCaseAttributes(LitElement) {
               })}
               @click=${() => (this.expanded = !this.expanded)}
             >
-              <div class="flex justify-between align-center">
+              <div class=${ccExpandable.title}>
```

Comment:
Shouldn't this be quoted?
```suggestion
              <div class="${ccExpandable.title}">
```
...or is that automagically fixed someway?

Reply:
don't know, It did work! but I added the quotes now anyway😅